ui/src/chatbot.tsx

```diff
@@ -32,44 +32,39 @@ const
     },
     msgContainer: {
       position: 'absolute',
-      top: 15,
+      top: 0,
       left: 0,
       right: 0,
-      bottom: 62, // Height of input box + padding.
+      bottom: 0,
       overflowY: 'auto',
-      padding: padding(0, 15),
+      padding: padding(0, 0),
+    },
+    msgWrapper: {
+      '&:first-child': { marginTop: important(px(0)) },
+      '&:last-child': { marginBottom: important(px(100)) },
+      display: 'flex',
+      justifyContent: 'center',
     },
     msg: {
+      width: '65ch',
+    },
+    userMsg: {
+      backgroundColor: cssVar('$neutralLighter'),
+    },
+    text: {
       display: 'inline-block',
-      backgroundColor: cssVar('$text'),
-      padding: 6,
-      borderRadius: 4,
       maxWidth: '65ch',
-      borderTopLeftRadius: 0,
       textAlign: 'left',
     },
-    userMsg: {
-      backgroundColor: cssVar('$themePrimary'),
-      borderTopLeftRadius: 4,
-      borderTopRightRadius: 0,
-    },
-    msgWrapper: {
-      '&:first-child': { marginTop: important(px(0)) },
+    textInput: {
+      position: 'absolute',
+      padding: 15,
+      bottom: 0,
+      left: 0,
+      right: 0,
+      backgroundImage: `linear-gradient(180deg,rgba(0,0,0,0),${cssVar('$card')} 45%)`
```

Comment:
Leftover?

Reply:
Nope. It makes a chat feed to fade at the bottom.

![Screenshot 2023-06-12 at 11 03 02](https://github.com/h2oai/wave/assets/23740173/ce82d3ae-5bc7-425f-98f9-617663dfe4d1)

![Screenshot 2023-06-12 at 11 03 50](https://github.com/h2oai/wave/assets/23740173/81d87f55-3e64-40d1-b93f-276c9468be26)

Reply:
Let's use a proper layout to make sure the scrollbar ends where it should - above the textbox, rather than continuing beyond. Same as before:

<img width="815" alt="image" src="https://github.com/h2oai/wave/assets/64769322/edbf04fe-d351-4d53-8266-0b5f473a19f1">